core-rust-bridge/src/main/java/com/radixdlt/statecomputer/commit/PrepareRequest.java

```diff
@@ -75,8 +76,11 @@ public record PrepareRequest(
     HashCode parentAccumulatorHash,
     List<PreviousVertex> previousVertices,
     List<RawNotarizedTransaction> proposed,
+    boolean isFallback,
     UInt64 epoch,
     UInt64 roundNumber,
+    List<ECDSASecp256k1PublicKey> gapRoundLeaderKeys,
```

Comment:
As per the other comment - we should be using the ComponentAddress rather than the public keys here. (because theoretically two validators in the set could share the same key).